src/test/java/org/apache/commons/collections4/bloomfilter/EnhancedDoubleHasherTest.java

```diff
@@ -43,39 +43,39 @@ protected int getHasherSize(Hasher hasher) {
     @Test
     public void testByteConstructor() {
         // single value become increment.
-        EnhancedDoubleHasher hasher = new EnhancedDoubleHasher( new byte[] { 1 } );
-        assertEquals( 0, hasher.getInitial() );
-        assertEquals( 0x01_00_00_00_00_00_00_00L, hasher.getIncrement() );
+        EnhancedDoubleHasher hasher = new EnhancedDoubleHasher(new byte[] { 1 });
```

Comment:
These formatting changes should not be part of this PR. I have applied formatting to master so you can rebase and should pick up the changes.